src/v0/destinations/pinterest_tag/utils.js

```diff
@@ -293,9 +293,11 @@ const processHashedUserPayload = (userPayload, message) => {
   });
   // multiKeyMap will works on only specific values like m, male, MALE, f, F, Female
   // if hashed data is sent from the user, it is directly set over here
-  processedHashedUserPayload.ge = [
-    message.traits?.gender || message.context?.traits?.gender || null,
-  ];
+  const gender = message.traits?.gender || message.context?.traits?.gender;
+  if (gender) {
+    processedHashedUserPayload.ge = [gender];
```

Comment:
1. Have we tested for other fields?
2. Please add a test case for this

Reply:
1. For other fields we are not sending null values
2. Test cases is there.
3. E2E [here](https://www.notion.so/rudderstacks/Test-Results-d400e26269c64df2bb718b4011230d75)